package.json

```diff
@@ -14,7 +14,7 @@
     "coding",
     "dev"
   ],
-  "author": "Ryan Fitzgerald",
```

Comment:
Should definitely leave his name in

Reply:
It does say it in the description field though

Reply:
I added his name back in (doesn't show up as added because it's in master), would normally click resolve conversation below